### DISCUSSION TITLE
How do I configure c++ debugging in github codespaces

### DISCUSSION THREAD

**Discussion by jujumumu** [community/community#45983] [Codespaces] [answered]
### Select Topic Area

Question

### Body

I tried running my project using codespaces for the first time using launch.json. I get "Configured debug type 'cppvsdbg' is not supported." What do I do?

**Comment by jge162** [comment, marked as answer]
Hello @jujumumu ->install the C/C++ extension for Visual Studio Code.

Then in your launch.json file, change the configured debug type to "cppdbg" instead of "cppvsdbg".

last, try running your project again and see if it works.
